pkg/raft/testdata/snapshot_succeed_via_app_resp_behind.txt

```diff
@@ -120,23 +116,12 @@ INFO 3 [commit: 11] restored snapshot [index: 11, term: 1]
 # current last index).
 stabilize 1
 ----
-> 1 receiving messages
-  3->1 MsgAppResp Term:1 Log:1/10 Rejected Hint:1/5 Commit:5
-  DEBUG 1 received MsgAppResp(rejected, hint: (index 5, term 1)) from 3 for index 10
-  DEBUG 1 decreased progress of 3 to [StateProbe match=0 next=6 sentCommit=5 matchCommit=5]
-  DEBUG 1 [firstindex: 11, commit: 12] sent snapshot[index: 12, term: 1] to 3 [StateProbe match=0 next=6 sentCommit=5 matchCommit=5]
-  DEBUG 1 paused sending replication messages to 3 [StateSnapshot match=0 next=13 sentCommit=12 matchCommit=5 paused pendingSnap=12]
-> 1 handling Ready
-  Ready:
-  Messages:
-  1->3 MsgSnap Term:1 Log:0/0
```

Comment:
Looks like this test now doesn't do what it intended. We need to coerce it into sending this snapshot again, and also update the comments above the commands.

Reply:
SGTM, the fix seems minimal.